agent/command/results_gotest.go

```diff
@@ -22,6 +22,11 @@ type goTestResults struct {
 	// a list of filename blobs to include
 	// e.g. "monitor.suite", "output/*"
 	Files []string `mapstructure:"files" plugin:"expand"`
+
+	// Optional, when set to true, causes this command to be skipped over without an error when
+	// no files are found to be parsed.
+	Optional bool `mapstructure:"optional" plugin:"expand"`
```

Comment:
I think Optional implies that all errors should be ignored, whereas here we just want to ignore empty output, so maybe OptionalOutput or etc would be more informative

Reply:
done